Api/ATM/ATM1/Request.php

```diff
@@ -83,18 +73,23 @@ public function getResponseClassName()
     {
         return Response::className();
     }
-
-    /**
-     * @param string $lightCertificate
-     * @param string $lightKey
-     */
-    public function cert($lightCertificate, $lightKey) {
+  
+  /**
+   * @param string $lightCertificate
+   * @param string $lightKey
+   * @param string $lightPass
+   */
+    public function cert($lightCertificate, $lightKey, $lightPass = '') {
```

Comment:
Something wrong with code formatting. What do you think about dropping "light" from var names and add to function? I think that it will be better. Just question, please don't touch names until @baibaratsky  will not answer